docs/source/performance/caching.md

```diff
@@ -198,6 +199,81 @@ const server = new ApolloServer({
 }));
 ```
 
+### Recommended starting usage
+
+You usually don't need to specify cache hints for every field in your schema. Instead, we recommend doing the following as a starting point:
+
+* For fields that should _never_ be cached, explicitly set `maxAge` to `0`.
+
+* Set a `maxAge` for every field with a resolver that _actually fetches data from a data source_ (such as a database or REST API). You can base the value of `maxAge` on the frequency of updates that are made to the relevant data.
+
+* Set `inheritMaxAge: true` for each other field that returns a non-scalar type.
+
+    * Note that you can only set `inheritMaxAge` [statically](#in-your-schema-static).
+
+### Example `maxAge` calculations
+
+Consider the following schema:
+
+```graphql
+type Query {
```

Comment:
I like how you put real concepts for this example instead of my foo etc. That said, while there's no way you could have guessed this since I left no breadcrumbs, I added these examples to the test suite described as doc examples, so it would be nice to update the tests too.  Grep for `inheritMaxAge docs examples`.  Or I can do it if you'd like.